subsys/llext/shell.c

```diff
@@ -11,28 +11,36 @@
 #include <zephyr/llext/elf.h>
 #include <zephyr/llext/llext.h>
 #include <zephyr/llext/buf_loader.h>
+#include <zephyr/llext/fs_loader.h>
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(llext_shell, CONFIG_LLEXT_LOG_LEVEL);
 
 #define LLEXT_LIST_HELP "List loaded extensions and their size in memory"
 
-#define LLEXT_LOAD_HEX_HELP								\
-	"Load an elf file encoded in hex directly from the shell input. Syntax:\n"	\
+#define LLEXT_LOAD_HEX_HELP                                                                        \
+	"Load an elf file encoded in hex directly from the shell input. Syntax:\n"                 \
```

Comment:
Nit: a few redundant whitespace changes in this file.

Reply:
This is how clang-format formats the code according to configuration provided in Zephyr. I decided to align all of them instead of formatting my change only